### PR TITLE
Support variable time windows for epochs metadata creation

### PR DESCRIPTION
**Pull request by hoechenberger**
This is for sometime after MNE 1.7 is out

### Before merging …

- [x] Changelog has been updated (`docs/source/changes.md`)

### CODE REVIEW

File: docs/source/v1.7.md.inc [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ### :new: New features & enhancements
 
+- Epochs metadata creation now supports variable time windows by specifying the names of events via [`epochs_metadata_tmin`][mne_bids_pipeline._config.epochs_metadata_tmin] and [`epochs_metadata_tmax`][mne_bids_pipeline._config.epochs_metadata_tmax]. (#873 by @hoechenberger)
```
Comment (hoechenberger) [inline docs/source/v1.7.md.inc:5]:
```suggestion
```

File: docs/source/v1.7.md.inc [hunk 2]
```diff
@@ -12,7 +13,9 @@
 
 [//]: # (- Whatever (#000 by @whoever))
 
-[//]: # (### :package: Requirements)
+### :package: Requirements
+
+- The minimum required version of MNE-Python is now 1.7.0.
```
Comment (hoechenberger) [inline docs/source/v1.7.md.inc:18]:
```suggestion
[//]: # (### :package: Requirements)
```

**Comment by hoechenberger** [comment]
Seems to be working – ready to merge!

**Comment by larsoner** [comment]
Thanks @hoechenberger

**Pull request merged**